source/magpie/index.d

```diff
@@ -378,12 +386,14 @@ public:
                     try
```

Comment:
You should really try to avoid exception handling in loops where possible. What is `next` expected to be? `int[]`? `string[]`? It would be good to put some constraint on what types it can be.

Reply:
Removed the try catch logic. Added a note telling assignment of one type to another will require an explicit conversion of Axis :+1: